pyproject.toml

```diff
@@ -22,3 +22,6 @@ isort = "^5.10.1"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[virtualenvs]
+in-project = true
```

Comment:
This is generally my default too, but do we need it for this functionality to work? I think it's fine to leave in either way since we're still pretty early on

Reply:
With the way the frontend is set up, this is needed since python-shell path to the python venv needs to be in the project. I think we should leave it in for now and when I start my work with packaging/distribution, this will change